Dockerfile

```diff
@@ -23,4 +23,4 @@ HEALTHCHECK --interval=30s --timeout=10s --retries=3 CMD curl -f http://localhos
 EXPOSE 3002
 
 # Start the Next.js app in production mode
-CMD ["npm", "run", "start"]
+CMD ["/usr/local/bin/docker-entrypoint.sh"]
```

Comment:
:thinking: - Is this working for you? If I remove the `command` override in my `docker-compose.yml`, the container seems to cycle for me :man_shrugging: .

Reply:
@rnathuji I was testing a fix to `exec /usr/local/bin/docker-entrypoint.sh -- exec syntax error`. And that line mistakenly was included in the MR. The issue was due to the fact that I build the image from a mac computer and ran it on linux.